shardingsphere-test/shardingsphere-rewrite-test/src/test/resources/scenario/sharding/case/select.xml

```diff
@@ -415,26 +415,26 @@
 
     <rewrite-assertion id="select_top_with_multiple_route_for_parameters_for_sqlserver" db-types="SQLServer">
         <input sql="SELECT * FROM (SELECT TOP(?) row_number() OVER (ORDER BY o.account_id) AS rownum_, o.account_id FROM t_account o WHERE o.account_id IN (100, 101)) AS row_ WHERE row_.rownum_ &gt; ?" parameters="110, 100" />
-        <output sql="SELECT * FROM (SELECT TOP(?) row_number() OVER (ORDER BY o.account_id) AS rownum_, o.account_id FROM t_account_0 o WHERE o.account_id IN (100, 101)) AS row_ WHERE row_.rownum_ &gt; ?" parameters="110, 0" />
-        <output sql="SELECT * FROM (SELECT TOP(?) row_number() OVER (ORDER BY o.account_id) AS rownum_, o.account_id FROM t_account_1 o WHERE o.account_id IN (100, 101)) AS row_ WHERE row_.rownum_ &gt; ?" parameters="110, 0" />
+        <output sql="SELECT * FROM (SELECT TOP(?) row_number() OVER (ORDER BY o.account_id) AS rownum_, o.account_id FROM t_account_0 o WHERE o.account_id IN (100, 101)) AS row_ WHERE row_.rownum_ &gt; ?" parameters="110, 100" />
```

Comment:
Hi @ThanoshanMV, the initial test case is correct. When we perform distributed paging query on multiple data nodes, we need to obtain the data starting from the 0 index, and then process it in the ShardingSphere result merge engine. 

I think a better way to solve the problem is to deploy an instance of SQLServer locally via docker, then create a sharding table and insert some data, and then execute paging query sql to investigate the cause of the problem. You can focus on investigating the `ShardingPaginationParameterRewriter` class.

Reply:
In addition, the final result is processed through the TopAndRowNumberDecoratorMergedResult class, you can also refer to it.

Reply:
Okay @strongduanmu, I'll check them.